coremltools/models/utils.py

```diff
@@ -1065,6 +1068,16 @@ def make_pipeline(*models):
     *models
         Two or more instances of ct.models.MLModel.
 
+    compute_units: coremltools.ComputeUnit
```

Comment:
We should also mention what compute units will be used for. 
Which is , that technically to make a pipeline spec, compute units is irrelevant, since thats just creation of a new combined proto spec. 
However, since this method returns an MLmodel object, which is a loaded Core ML model, compute unit refers to the set of units that model is loaded for.

Reply:
> We should also mention what compute units will be used for.

Good idea. This information also wasn't included in `ct.convert` or `MLModel`. So I've added it there too.

> Which is , that technically to make a pipeline spec, compute units is irrelevant, since thats just creation of a new combined proto spec.
> However, since this method returns an MLmodel object, which is a loaded Core ML model, compute unit refers to the set of units that model is loaded for.

I'm not really understanding this part. If we my change didn't address this, please clarify.